php_phongo.c

```diff
@@ -1638,30 +1613,16 @@ static void php_phongo_cursor_iterator_rewind(zend_object_iterator *iter TSRMLS_
 {
 	php_phongo_cursor_iterator *cursor_it = (php_phongo_cursor_iterator *)iter;
 	php_phongo_cursor_t        *cursor = cursor_it->cursor;
+	const bson_t               *doc;
 
 	php_phongo_cursor_free_current(cursor);
 	cursor_it->current = 0;
```

Comment:
I have two more tests I can add to the PR tomorrow. Basically, I want to iterate through the Cursor with a `foreach` and call `rewind()` at each iteration and assert that it's a NOP (both for altering the current element *and* the index). Since we can't very well rewind earlier into the BSON buffer (especially after a getmore overwrites a previous batch), I'd just like to test that rewinding doesn't change the value or key progression.